sdk/cargo-build-sbf/src/main.rs

```diff
@@ -562,10 +565,16 @@ fn build_sbf_package(config: &Config, target_directory: &Path, package: &cargo_m
     env::set_var("OBJCOPY", llvm_bin.join("llvm-objcopy"));
 
     let rustflags = env::var("RUSTFLAGS").ok();
-    let rustflags = rustflags.as_deref().unwrap_or_default();
+    let mut rustflags = Cow::Borrowed(rustflags.as_deref().unwrap_or_default());
     if config.remap_cwd {
-        let rustflags = format!("{} -Zremap-cwd-prefix=", &rustflags);
-        env::set_var("RUSTFLAGS", &rustflags);
+        rustflags = Cow::Owned(format!("{} -Zremap-cwd-prefix=", &rustflags));
+    }
+    if config.debug {
+        // Replace with -Zsplit-debuginfo=packed when stabilized.
+        rustflags = Cow::Owned(format!("{} -g", &rustflags));
```

Comment:
@jawilk has pointed out that `-g` might need `RUSTFLAGS="-C target-feature=+reloc-abs64"` too. This compiler feature was added by @alessandrod in https://github.com/solana-labs/llvm-project/commit/fb7188bcf651fdaa2d2c522f4b7444e2c574a822 and is enabled by default in `sbfv2`, I think.

Does this prevent us from generating correct DWARF sections for bpf and sbf? 😦

Reply:
Fixed, feature is now disabled by default